src/client/clientlauncher.cpp

```diff
@@ -537,19 +537,6 @@ bool ClientLauncher::launch_game(std::string &error_message,
 void ClientLauncher::main_menu(MainMenuData *menudata)
 {
 	bool *kill = porting::signal_handler_killstatus();
-	video::IVideoDriver *driver = RenderingEngine::get_video_driver();
-
-	infostream << "Waiting for other menus" << std::endl;
-	while (RenderingEngine::get_raw_device()->run() && !*kill) {
-		if (!isMenuActive())
-			break;
-		driver->beginScene(true, true, video::SColor(255, 128, 128, 128));
```

Comment:
Doesn't the other change already fix the bug?
I'd rather leave this code in.

Reply:
yes, the other change is enough.